catatonit.c

```diff
@@ -512,6 +512,14 @@ int main(int argc, char **argv)
 		debug("pid1 (%d) spawned: %s", pid1, argv[0]);
 	}
 
+	/*
+	 * Switch to / explicitly, to work around a known podman issue where podman
+	 * runs "catatonit -P" in a subdirectory which causes umount to fail
+	 * because the directory is pinned by catatonit. Ignore errors since we
+	 * don't care about the cwd anyway.
+	 */
+	(void) chdir("/");
```

Comment:
```suggestion
	(void) chdir("/var/lock");
```
I'm suggesting `/var/lock` as a slightly more principled place to have the cwd open-file exist in the hierachy.  See https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard for hints at a rationale.

Reply:
We aren't storing a lock file though, this is just our current directory.

`/` makes the most sense because inits generally run in `/` and it's the only path that we can be in that won't keep unneeded mounts busy.

Reply:
Yes, I know it's just the cwd file descriptor.  I made the `/var/lock` directory suggestion because my experience with this problem was that it locked my temporary mount and the `/var/lock` notes include "Files keeping track of resources currently in use" which is the closest description in the lists of semantics for directories that matched my experience....  But, you all have far more perspective and insight on what other developers will expect and understand.